### PR TITLE
display service dependencies in service details

### PR DESCRIPTION
**Pull request by jsladerman**
added table to list name and status of service dependencies

### CODE REVIEW

File: assets/src/components/cd/services/ServiceDependenciesColumns.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { createColumnHelper } from '@tanstack/react-table'
+
+const columnHelper = createColumnHelper<Nullable<ServiceDependency>>()
+
+const ColName = columnHelper.accessor('name', {
+  id: 'name',
+  header: 'Name',
+})
+
+const ColStatus = columnHelper.accessor('status', {
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/ServiceDependenciesColumns.tsx:11]: I think we should render the status as a chip with appropriate color.  Also, it's realistically only set as either nil or `HEALTHY` at the moment so there'd need to be ternary logic like `color={status === 'HEALTHY' ? 'success' : 'warn'}`

Comment (jsladerman) [reply assets/src/components/cd/services/ServiceDependenciesColumns.tsx:12]: updated

File: assets/src/components/cd/services/ServiceDependenciesColumns.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  id: 'status',
+  header: 'Status',
+  cell: ({ getValue }) => (
+    <div>
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/ServiceDependenciesColumns.tsx:16]: is the parent div needed?

File: assets/src/components/cd/services/ServiceDependenciesColumns.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <Chip
+        size="large"
+        severity={getValue() === 'HEALTHY' ? 'success' : 'warning'}
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/ServiceDependenciesColumns.tsx:19]: ideally we use the actual codegened enum here, but can't remember how to import it

**Review by michaeljguarino — APPROVED** [review]

File: assets/src/components/cd/services/ServiceDependenciesColumns.tsx (hunk 3 above)
Comment (jsladerman) [reply assets/src/components/cd/services/ServiceDependenciesColumns.tsx]: yeah good call. typescript was still throwing the warning if the checked string wasn't part of the enum but I went ahead and imported it

**Pull request merged**